okhttp/src/commonJvmAndroid/kotlin/okhttp3/OkHttpClient.kt

```diff
@@ -312,6 +312,7 @@ open class OkHttpClient internal constructor(
    * Creates an [Address] of out of the provided [HttpUrl]
    * that uses this client’s DNS, TLS, and proxy configuration.
    */
+  @Deprecated("Not intended to be public API on OkHttpClient")
```

Comment:
Why not? I think this is out of scope for this PR.

Reply:
Probably a bug that it’s here. But I don’t like deprecating stuff unless we’re going to remove it later, and I definitely don’t like removing stuff later. Ultimately I believe so much in this thing that I never want to break APIs, and we should have a stronger reason to deprecate something than ‘we don’t like it anymore’.

https://www.youtube.com/watch?v=oyLBGkS5ICk